signwriting/visualizer/test_visualize.py

```diff
@@ -55,10 +54,9 @@ def test_image_small_box_is_corrected(self):
         assert_array_equal(np.array(image), np.array(reference_image))
 
     def test_image_with_line_and_embedded_color(self):
```

Comment:
this test is now problematic as it tests too many things at once: line color, and transparent fill.
the name suggests it tests `embedded_color` but it doesn't

needs at least three tests:
1. embedded_color
2. line_color
3. fill_color
4. ideally, line and fill
5. ideally, line and embedded